dartagnan/src/main/java/com/dat3m/dartagnan/wmm/Wmm.java

```diff
@@ -39,7 +39,7 @@ public static class Config {
 
     // These relations are part of every memory model (even the "empty" one) because they are
     // necessary to specify the anarchic program semantics.
-    public final static ImmutableSet<String> ANARCHIC_CORE_RELATIONS = ImmutableSet.of(CO, RF, RMW);
+    public final static ImmutableSet<String> ANARCHIC_CORE_RELATIONS = ImmutableSet.of(CO, RF, LXSX);
```

Comment:
Shouldn't `AMO` be here?

Reply:
`LXSX` is correct here. This relation has the dynamic pairing computation for LX and SX.
AMO events on the other hand are statically paired and there is also no notion of failure (unlike SX), so the AMO relation is completely static.